src/Stratis.Bitcoin.Networks/Deployments/StratisBIP9Deployments.cs

```diff
@@ -0,0 +1,45 @@
+﻿using NBitcoin;
+
+namespace Stratis.Bitcoin.Networks.Deployments
+{
+    /// <summary>
+    /// BIP9 deployments for the Stratis network.
+    /// </summary>
+    public class StratisBIP9Deployments : BIP9DeploymentsArray
+    {
+        // The position of each deployment in the deployments array.
+        public const int TestDummy = 0;
+        public const int CSV = 1;
```

Comment:
As it happens, CSV is not activated in StratisX, so you can remove this.

Reply:
Lets keep CSV but not give it any activation dates, we have some intentions to maybe activated it in the future.